FirstTask.py

```diff
@@ -0,0 +1,230 @@
+        if status_1 and status_2:
+            if response_1.draw_money(amount):
+                response_2.deposit_bill(amount)
+                return True
+        else:
+            return False
+    
+    def compare(self, current_account, other_account):
+        status_1, response_1 = self.find_account(current_account)
+        status_2, response_2 = self.find_account(other_account)
+        if status_1 and status_2:
+            if response_1.check_balance() > response_2.check_balance():
+                return True, 1
+            if response_1.check_balance() < response_2.check_balance():
+                return True, -1
+            if response_1.check_balance() == response_2.check_balance():
+                return True, 0
+        else:
+            return False, f"{other_account} Account Not Found!"
+    
+   
+    def account_details(self, account_number):
+        status,response = self.find_account(account_number)
+        if status:
+            response.account_details()
+        else:
+            return False
+   
+    def logout(self):
+        self.logged_in = None
+        return self.logged_in
+
+    def is_logged_in(self):
+        if self.logged_in:
+            return True
+        return False
+
+def run():
+    account_number_id = 1000 # Wrirting here just to print account number for user after account creation 
+    bank = Bank()
+    while(1):
+        option = int(input("1) Create Account \n2) Login \n3) Find Account \n4) Total Accounts\n0) Exit \nChoose:"))
+        if option == 1:
+            sys('clear')
+            account_number_id += 1
+            name = input("Enter Full Name: ")
+            pass_code = int(input("Enter Passcode "))
+            while(1):
+                ACCOUNT_TYPES = ["Current","Saving","Business"]
+                account_select = int(input("Select Account Type from following:\n1) Current \n2) Saving \n3) Business\n"))
+                if account_select == 1:
+                    account_type = ACCOUNT_TYPES[1]
+                    break
+                elif account_select == 2:
+                    account_type = ACCOUNT_TYPES[2]
+                    
+                    break
+                elif account_select == 3:
+                    account_type = ACCOUNT_TYPES[3]
+                    break
+                else:
+                    print("\nInvalid Account Type\n")
+            bank.create_account(name, pass_code,account_type)
+            print(f"Account Created Successfully\nYour Account Number is {account_number_id}\n\n")
+            input("\nPress Any Key to Continue\n")
+            sys("clear")
+        
+        elif option == 2:
+            if bank.is_logged_in() == False:
+                account_number = int(input("Enter Account Number: "))
+                code = int(input("Enter Passcode: "))
+                log = bank.login(account_number, code)
+
+            if bank.is_logged_in:

```

Comment:
`is_logged_in` is a method not an attribute. This would throw an error. Try to test each case after implementing.